### DISCUSSION TITLE
Optional Low-Cut Setting?

### DISCUSSION THREAD

**Discussion by Helondeth** [jamulussoftware/jamulus#1458] [Ideas]
English: 
Hello all,
During rehearsals with my choir I had an idea regarding the software.
Not all of them have high quality equipment, so that some singers have a low frequent hum.
Would it be possible to add an optional low-cut (e.g. from 75hz) to the software to filter this out?

German:
Hallo zusammen,
bei den Proben mit meinem Chor kam mir eine Idee bzgl. der Software. 
Nicht alle besitzen hoch qualitatives Equipment, so dass bei manchen Sängern ein tief frequentes Brummen entsteht.
Wäre es möglich einen optionalen Low-Cut (bspw. ab 75hz) in die Software einzubauen um dieses herauszufiltern?

Greetings

**Comment by ann0see** [comment]
Hallo!

Ich denke, dass es gut wäre, wenn du deine Idee auf Englisch übersetzt postest, damit mehr Nutzer dich verstehen!

Aus der Erfahrung würde ich sagen, dass dein Feature erst einmal (heftig) diskutiert wird. Guck dir die Source Code Guidelines, und das KISS (keep it simple and stupid) Prinzip an. Ich denke, dass EQ und Ähnliches eher nicht speziell integriert werden würde. Eventuell VST Support, aber da würde ich eine andere Diskussion vorschlagen (https://github.com/jamulussoftware/jamulus/discussions/1294)

**Reply by dcorson-ticino-com** [reply]
Ups, ich habe es gelesen und nicht mal registriert, das es auf Deutsch und nicht English ist.
